### DISCUSSION TITLE
How to Ignore the Month and Day Fields in a cron

### DISCUSSION THREAD

**Discussion by gadelkareem** [pallets-eco/croniter#189] [Announcements]
I need to parse a cron expression but ignore the month and day of the month fields, limiting the cron to only the current week. Additionally, I need to ignore formats such as '0 0 * * 5#3,L5'. Is this possible?

**Comment by potiuk** [comment]
I believe only when you create a custom post-filtering, but If you provide some more concrete examples what you want to achieve maybe someone will be able to help. Converted into a discussion - this is really a question, not issue.
